gerrit/ref-update

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+# Copyright 2019-present Open Networking Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Gerrit ref-update hook for cla-manager
+#
+# Gerrit Hook Documentation:
+# https://gerrit.googlesource.com/plugins/hooks/+/master/src/main/resources/Documentation/hooks.md
+
+from __future__ import absolute_import
+
+import argparse
+import json
+import logging
+import re
+import sys
+import urllib.request
+
+# Edit this variable with the URL of the CLA manager instance
+CLA_MANAGER_URL = "https://cla-manager.example/check-cla"
```

Comment:
should we source this from an environment variable?

Reply:
This is run by Gerrit as a hook, which doesn't (to my knowledge) have the ability to set environmental variables.